cli/utils/calculate-rewards.ts

```diff
@@ -115,9 +126,10 @@ function updateBalanceByBlock(
   account: string,
   blockNumber: number
 ) {
-  if (state.attestationCompletions[account] >= 3) {
-    const balancesByBlock = state.balancesByBlock[account]
-    if (balancesByBlock.balances.length > 0) {
+  // @ts-ignore
+  if (accountIsVerified(state, account)) {
+    let balancesByBlock = state.balancesByBlock[account]
```

Comment:
Can this be a const?

Reply:
you know.....as you can see above I had it a const, but I renamed to let since I'm pushing to it later on. I was surprised that compiled in the first place, but maybe I'm missing something...*should* it be const??

Reply:
It's okay to be const since the object it's pointing to is never changed, but it still lets you manipulate the object (push in this case).